src/components/profile/profile.pw.tsx

```diff
@@ -0,0 +1,184 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import ProfileComponent from "./component.test-pw";
+import { CHARACTERS } from "../../../playwright/support/constants";
+
+import {
+  emailPreview,
+  namePreview,
+  profilePreview,
+  initialPreview,
+} from "../../../playwright/components/profile";
+
+import {
+  portraitImage,
+  portraitInitials,
+} from "../../../playwright/components/portrait";
+
+import { PROFILE_SIZES } from "../../../src/components/profile/profile.config";
+
+import {
+  checkAccessibility,
+  containsClass,
+} from "../../../playwright/support/helper";
+import { ProfileProps } from "./profile.component";
+
+const testImage =
+  "https://www.gravatar.com/avatar/05c1c705ee45d7ae88b80b3a8866ddaa?s=24&d=404";
+
+const imageURLs = [
+  "https://avataaars.io/?avatarStyle=Transparent&topType=LongHairStraight&accessoriesType=Blank&hairColor=BrownDark&facialHairType=Blank&clotheType=BlazerShirt&eyeType=Default&eyebrowType=Default&mouthType=Default&skinColor=Light",
+  "https://www.gravatar.com/avatar/05c1c705ee45d7ae88b80b3a8866ddaa?s=24&d=404",
+];
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+
+test.describe("Prop checks for Profile component", () => {
+  test(`should render with className`, async ({ mount, page }) => {
+    await mount(<ProfileComponent className="profile-playwright-classname" />);
+
+    await containsClass(profilePreview(page), "profile-playwright-classname");
+  });
+
+  testData.forEach((email) => {
+    test(`should render with email prop is passed as ${email}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ProfileComponent email={email} src={testImage} />);
+
+      await expect(emailPreview(page)).toHaveText(email);
+    });
+  });
+
+  testData.forEach((name) => {
+    test(`should render with name prop is passed as ${name}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ProfileComponent name={name} src={testImage} />);
+
+      await expect(namePreview(page)).toHaveText(name);
+    });
+  });
+
+  ([
+    [PROFILE_SIZES[0], 22],
+    [PROFILE_SIZES[1], 30],
+    [PROFILE_SIZES[2], 38],
+    [PROFILE_SIZES[3], 54],
+    [PROFILE_SIZES[4], 70],
+    [PROFILE_SIZES[5], 102],
+    [PROFILE_SIZES[6], 126],
+  ] as [ProfileProps["size"], number][]).forEach(([size, heightAndWidth]) => {
+    test(`should render with size prop is passed as ${size}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ProfileComponent size={size} />);
+
+      await expect(initialPreview(page)).toHaveCSS(
+        "height",
+        `${heightAndWidth}px`
+      );
+      await expect(initialPreview(page)).toHaveCSS(
+        "width",
+        `${heightAndWidth}px`
+      );
+    });
+  });
+
+  ["DJ", "SF"].forEach((passInitials) => {
+    test(`should render with correct width and height, when initials prop is passed as ${passInitials}`, async ({
```

Comment:
**issue**: this still needs to be addressed, imo this test is redundant as we're only testing one size (Medium) when the initials prop is passed. Instead we should be replicating the test above if we decide to keep it in. 

Also a new test needs to be made which achieves what the original cypress test was supposed to be doing, which is passing a name, no initials and ensuring initials are generated anyways due to our built in functionality `function acronymize(str?: string) {
  if (!str) return "";
  const matches = str.match(/\b\w/g);
  if (!matches) return "";
  return matches.join("");
}`